src/utils.js

```diff
@@ -39,7 +39,7 @@ export function decorateColumn(column, defaultColumn, parent, depth, index) {
 
   column = {
     Header: ({ id }) => id,
-    Cell: ({ value }) => value,
+    Cell: ({ value }) => value ? value : '',
```

Comment:
How about `typeof value !== 'undefined' ? value : ''`?

Reply:
Off course, you right ! With few reflection my pseudo fix isn't a really one :-(